### PR TITLE
Ecosystem Mem0

### PR DESCRIPTION
**Pull request by marklysze**
## Why are these changes needed?

Update to mem0 docs created by @Dev-Khant from forked repo, [PR 3492](https://github.com/microsoft/autogen/pull/3492) and [PR 3389](https://github.com/microsoft/autogen/pull/3389).

Thanks to @Dev-Khant

## Checks

- [ ] I've included any doc changes needed for https://autogen-ai.github.io/autogen/. See https://autogen-ai.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [X] I've made sure all auto checks have passed.

### CODE REVIEW

File: notebook/lats_search.ipynb [hunk 1]
```diff
@@ -1036,15 +1034,6 @@
...
  "metadata": {
-  "front_matter": {
-    "description": "Language Agent Tree Search.",
-    "tags": [
-        "LATS",
-        "search",
-        "reasoning",
-        "reflection"
-    ]
-  },
```
Comment (sonichi) [inline notebook/lats_search.ipynb]: Can we keep these metadata?

File: notebook/agentchat_memory_using_mem0.ipynb [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "di1opljX6sQT"
+   },
+   "source": [
+    "This notebook demonstrates an intelligent customer service chatbot system that combines:\n",
+    "\n",
+    "- PyAutoGen for conversational agents\n",
```
Comment (sonichi) [inline notebook/agentchat_memory_using_mem0.ipynb]:
```suggestion
    "- AutoGen for conversational agents\n",
```

File: notebook/autobuild_agent_library.ipynb
Comment (sonichi) [inline notebook/autobuild_agent_library.ipynb:1]: Is the change to this notebook necessary?
cc @LeoLjl @LinxinS97

Comment (marklysze) [reply notebook/autobuild_agent_library.ipynb:1]: Hmmm, looking at the notebook vs the website, they look identical, I'll remove the change from the PR.

**Review by Dev-Khant — APPROVED** [review]

**Review by Hk669 — APPROVED** [review]
looks good to me. thanks @marklysze .

thanks @Dev-Khant for contributing.

**Pull request merged**